test/run-openshift

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+#
+# Test the Postgresql image in OpenShift.
+#
+# IMAGE_NAME specifies a name of the candidate image used for testing.
+# The image has to be available before this script is executed.
+#
+
+THISDIR=$(dirname ${BASH_SOURCE[0]})
+
+source ${THISDIR}/test-lib-openshift.sh
+
+set -exo nounset
+
+function check_postgresql_os_service_connection() {
+  local util_image_name="${1}" ; shift
```

Comment:
Nit:please use just "foo=$baz" syntax.  Looks much nicer, the quotes and brackets are not needed in this case.